workers.go

```diff
@@ -167,15 +182,41 @@ func (api *API) ListWorkerScripts() (WorkerListResponse, error) {
 // UploadWorker push raw script content for your worker
 //
 // API reference: https://api.cloudflare.com/#worker-script-upload-worker
-func (api *API) UploadWorker(requestParams *WorkerRequestParams, data string) (WorkerScriptResponse, error) {
+func (api *API) UploadWorker(requestParams *WorkerRequestParams, data string, metadataParams *WorkerMetadataParams) (WorkerScriptResponse, error) {
```

Comment:
there is no requirement to upload metadata and changing the public facing interface would require people to update their code to pass in a second nil parameter. 

What do you think about updating this PR to keep this function as is and adding a new `UploadWorkerWithResourceBindings` function to so we don't change API?